colossus/src/main/scala/colossus/controller/Controller.scala

```diff
@@ -297,8 +300,11 @@ extends ConnectionHandler {
           }
           currentTrigger = Some(trigger)
         }
-        case Failure(reason) => {
-          //todo: what to do here?
+        //TODO: are the following 2 cases really exceptions..seems like they are part of the normal control flow from a controller's POV
```

Comment:
There is an issue #82 to address those, and that stuff can go into another branch I think.